CHANGES/7740.bugfix

```diff
@@ -0,0 +1 @@
+Sync more than 10 versions per collection
```

Comment:
How about "Fixed bug where only 10 collections were being synced in some cases"

Reply:
much better!